rules/android_application/BUILD

```diff
@@ -41,5 +41,6 @@ filegroup(
     name = "merge_feature_manifests.par",
     srcs = [":merge_feature_manifests"],
     output_group = "python_zip_file",
+    visibility = ["//visibility:public"],
```

Comment:
Does this need to be public, or is repo-level visibility (i.e. `//:__subpackages__`) enough?